### PR TITLE
Allow configuration of ingress

### PR DESCRIPTION
**Pull request by jbeard6**
We use ingress, but not Nginx. Also, we need to use a specific hostname and labels.

This allows us to configure these values in our, ahem, values.

### CODE REVIEW

**Comment by jbeard6** [comment]
I updated this PR as we needed to completely replace the annotations. This method is in-keeping with the approach in the web service service and will work for us.

File: templates/tonic-web-server-ingress.yaml [hunk 1]
```diff
@@ -14,6 +18,13 @@ metadata:
...
     nginx.ingress.kubernetes.io/proxy-send-timeout: "3600" # 1 hour
     nginx.ingress.kubernetes.io/proxy-body-size: "0" # disable body size constraint
+  {{- end }}
+  {{- if .Values.tonicai.ingress.labels }}
+  labels:
+    {{- with .Values.tonicai.ingress.labels }}
+    {{ toYaml . | nindent 4 }}
+    {{- end }}
+  {{- end }}
 spec:
```
Comment (anr-tonic) [inline templates/tonic-web-server-ingress.yaml:28]:
```suggestion
spec:
  {{- if .Values.tonicai.ingress.class }}
  ingressClassName: {{ .Values.tonicai.ingress.class }}
  {{- end }}
```

Comment (anr-tonic) [reply templates/tonic-web-server-ingress.yaml:28]: Looks good other wise, I'll merge after this change.

Comment (jbeard6) [reply templates/tonic-web-server-ingress.yaml:28]: @anr-tonic This has been added.

**Review by anr-tonic — APPROVED** [review]

**Comment by anr-tonic** [comment]
Looks like some conflicts, will merge after those are resolved as well

**Comment by jbeard6** [comment]
@anr-tonic Conflicts resolved. 👍

**Pull request merged**